app/assets/stylesheets/components/_option-select.scss

```diff
@@ -82,7 +82,6 @@
 
 .app-c-option-select__container {
   position: relative;
-  max-height: 200px;
   overflow-y: auto;
```

Comment:
I think we can remove the overflows along with this?

Reply:
Thanks @maxgds - this wasn't quite ready for eyes just yet - i'm getting errors rendering `http://static.dev.gov.uk/templates/header_footer_only.html.erb` and wanted to see this on an environment. 

It's good to know the review request comes through once tagged rather than when marked ready.

Reply:
Ah, my mistake, sorry.